web/src/engine/osk/src/banner/bannerView.ts

```diff
@@ -86,6 +86,7 @@ export class BannerView implements OSKViewComponent {
         let prevBanner = this.currentBanner;
         this.currentBanner = banner;
         this.bannerContainer.replaceChild(banner.getDiv(), prevBanner.getDiv());
+        prevBanner.shutdown();
```

Comment:
Is this a change in behaviour?

Reply:
I was surprised when I realized that we never explicitly "destroyed" the banner's gesture engine instance whenever it was swapped out.  I _believe_ that standard garbage-collection would handle things properly anyway - in which case there is no net change in behavior.  If I'm wrong, and GC isn't able to handle things without this change, this would then help prevent a memory leak when repeatedly swapping among the banner types.

Reply:
Gotcha. It'd probably be better process-wise to split it into a separate PR, and keep this one to shuffling the deck chairs. Appreciate that that feels like busywork.

Reply:
This is now implemented in #11599.